src/transformers/image_utils.py

```diff
@@ -579,9 +579,15 @@ def normalize(self, image, mean, std, rescale=False):
             import torch
 
             if not isinstance(mean, torch.Tensor):
-                mean = torch.tensor(mean)
```

Comment:
as a general note: this class isn't used anymore so changing this won't have any effect. Let's keep though to be consistent with the other mixins